### PR TITLE
:fire: Drop usage of order field.

### PR DESCRIPTION
**Pull request by ghislainfourny**

### CODE REVIEW

File: queries/modules/io/28/modules/xbrl/networks.jq [hunk 1]
```diff
@@ -135,10 +135,11 @@ declare %private function networks:merge-trees($trees as object*) as object*
 {
     for $trees in $trees
     group by $name := $trees.Name
+    count $order
```
Comment (dknochen) [inline queries/modules/io/28/modules/xbrl/networks.jq]: there is already a "count $count" ...

File: queries/modules/io/28/modules/xbrl/networks.jq [hunk 2]
```diff
@@ -135,10 +135,11 @@ declare %private function networks:merge-trees($trees as object*) as object*
...
     let $order :=
         if($trees.PreferredLabelRole = "http://www.xbrl.org/2003/role/totalLabel")
         then 1000000
-        else min($trees.Order)
+        else $order
```
Comment (dknochen) [inline queries/modules/io/28/modules/xbrl/networks.jq]: I don't understand what this does? Could we add a comment, that explains it? thank you

File: queries/modules/io/28/modules/xbrl/networks.jq (hunk 1 above)
Comment (ghislainfourny) [inline queries/modules/io/28/modules/xbrl/networks.jq]: Yes, but it's overriden with the let and the order by, so the second one is not superfluous.

File: queries/modules/io/28/modules/xbrl/networks.jq (hunk 2 above)
Comment (ghislainfourny) [inline queries/modules/io/28/modules/xbrl/networks.jq]: I added some comments. This code simply pushes totals to the back.

**Comment by dknochen** [comment]
very good :+1: If tests pass then I approve... :)

**Comment by ghislainfourny** [comment]
They should in this round ;-) The merged components were a bit reshuffled, as expected (the algorithm no longer uses the original order fields).

**Comment by ghislainfourny** [comment]
Old:

http://rendering.secxbrl.info/#?url=http:%2F%2Fedinet.28.io:80%2Fv1%2F_queries%2Fpublic%2Fapi%2Fspreadsheet-for-component.jq%3Ftoken%3Dc3049752-4d35-43da-82a2-f89f1b06f7a4%26tag%3DNIKKEI%26fiscalYear%3D2014%26fiscalPeriod%3DQ1%26filingKind%3DEDINET%26role%3Dhttp%253A%252F%252Fdisclosure.edinet-fsa.go.jp%252Frole%252Fjppfs%252Frol_YearToQuarterEndConsolidatedStatementOfIncome%26eliminate%3Dtrue%26language%3Den%26format%3Djson

New:

http://rendering.secxbrl.info/#?url=http:%2F%2Fedinet-dev4.28.io:80%2Fv1%2F_queries%2Fpublic%2Fapi%2Fspreadsheet-for-component.jq%3Ftoken%3Dfoobar%26tag%3DNIKKEI%26fiscalYear%3D2014%26fiscalPeriod%3DQ1%26filingKind%3DEDINET%26role%3Dhttp%253A%252F%252Fdisclosure.edinet-fsa.go.jp%252Frole%252Fjppfs%252Frol_YearToQuarterEndConsolidatedStatementOfIncome%26eliminate%3Dtrue%26language%3Den%26format%3Djson

As you see, this change has no semantic meaning as the merge order was already arbitrary.

**Pull request merged**